### DISCUSSION TITLE
Any way to go to the change from a commit's commit view

### DISCUSSION THREAD

**Discussion by printfdebugging** [NeogitOrg/neogit#1237] [Q&A] [answered]
The commit view is nice, where I can see all the changes, but many a times, I have to go to that change, to study it, see other
functions etc involved, maybe edit it (in cases where I am submitting the patches for review, and have to commit the changes).

Is there any way to open the file with the cursor on that change?

**Comment by CKolkey** [comment, marked as answer]
There isn't, but it should be pretty doable to open the buffer in a read-only context.. not sure about editing it further, though. You could check out the commit from that buffer (`bb` and choose the sha) and modify that, though.
